src/AdiBags_ByExpansion_TBC/professions/gathering/mining.lua

```diff
@@ -2,7 +2,20 @@ local AddonName, AddonTable = ...
 
 AddonTable.mining = {
     -- Ore
+    23424, -- Fel Iron Ore
     23425, -- Adamantite Ore
     23427, -- Eternium Ore
+    23426, -- Khorium Ore
+
+    -- Bars 
+    23445, -- Fel Iron Bar
+    23446, -- Adamantite Bar
+    23447. -- Eternium Bar
```

Comment:
A couple of typo's here, periods need to be comma's, but otherwise everything else in this PR is amazing :)

Reply:
I fixed some. Typos happen, especially 'cus I do this while playing hehe.